src/main/app/common/frequency/frequency.ts

```diff
@@ -1,9 +1,10 @@
 import { PaymentSchedule } from 'claims/models/response/core/paymentSchedule'
+import { PaymentFrequency } from 'claims/models/response/core/paymentFrequency'
 
 export class Frequency {
   static readonly WEEKLY = new Frequency(['WEEK', 'EACH_WEEK'], 52 / 12, 1, 'Each week')
-  static readonly TWO_WEEKLY = new Frequency(['TWO_WEEK', 'EVERY_TWO_WEEKS'], 52 / 12 / 2, 2, 'Every 2 weeks')
-  static readonly FOUR_WEEKLY = new Frequency(['FOUR_WEEK', 'EVERY_FOUR_WEEKS'], 52 / 12 / 4, 4, 'Every 4 weeks')
+  static readonly TWO_WEEKLY = new Frequency(['TWO_WEEK', 'EVERY_TWO_WEEKS', 'TWO_WEEKS'], 52 / 12 / 2, 2, 'Every 2 weeks')
+  static readonly FOUR_WEEKLY = new Frequency(['FOUR_WEEK', 'EVERY_FOUR_WEEKS', 'FOUR_WEEKS'], 52 / 12 / 4, 4, 'Every 4 weeks')
```

Comment:
Is there any actual requirement to keep `TWO_WEEK` and `FOUR_WEEK`? Can't they just be replaced?

Reply:
I agree, I don't think there is and it could be brought it in-line with values already present